node/pod.go

```diff
@@ -213,7 +216,9 @@ func (pc *PodController) enqueuePodStatusUpdate(ctx context.Context, q workqueue
 		if obj, ok := pc.knownPods.Load(key); ok {
 			kpod := obj.(*knownPod)
 			kpod.Lock()
-			kpod.lastPodStatusReceivedFromProvider = pod
+			if !cmp.Equal(kpod.lastPodStatusReceivedFromProvider, pod) {
```

Comment:
If cmp.equal, then we don't need to call q.AddRateLimited below.

Reply:
Yes, thanks

Reply:
Done, PTAL